feat: remove outdated version create message

kyle-blair opened the pull request:
### What does this PR do?

Removes an old message about code coverage requirements for promoting unlocked package versions. The information is several years old.

### What issues does this PR fix or reference?

It was a simple enough change that I didn't bother opening an issue.

shetzel commented:
@kyle-blair - Turns out there is a test that looks for this message.  If you'd like to get credit for this PR (rather than me making the change and getting the credit) can you remove that check here: plugin-packaging/test/commands/package/versionPromoteUpdate.nut.ts:51

kyle-blair commented:
> @kyle-blair - Turns out there is a test that looks for this message. If you'd like to get credit for this PR (rather than me making the change and getting the credit) can you remove that check here: plugin-packaging/test/commands/package/versionPromoteUpdate.nut.ts:51

Sorry about that @shetzel, I had issues with the yarn install and therefore didn't notice the test. I rebased to upstream main and squashed the test update into a single commit. Is the unsigned commit an issue? I don't have signing set up on this computer yet, but I can replace the commit with a signed one tomorrow if necessary. Thanks for taking a look!
